doc/index.rst

```diff
@@ -1,12 +1,19 @@
+.. Orange3-Associate documentation master file, created by
+   sphinx-quickstart on Fri Jun  3 12:00:36 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
 
-Orange-Associate scripting documentation
-========================================
+Welcome to Orange3-Associate documentation!
+===========================================
+
+Widgets
+-------
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
-.. automodule:: fpgrowth
-   :members:
+   widgets/associationrules.rst
+   widgets/frequentitemsets.rst
```

Comment:
Great, this links the two widget documentations. However, this package can be really efficiently used from a script (possibly the best Python implementation of association rules mining :stuck_out_tongue: ), but the documentation from fpgrowth module is no longer linked/shown.